peachjam/js/components/LegislationTable/index.vue

```diff
@@ -157,16 +157,12 @@
           </template>
           <div
             v-else
-            class="legislation-table__row"
+            class="p-2 text-center"
           >
-            <div
-              class="indent"
-            />
-            <div
-              class="column"
-            >
-              No documents found
-            </div>
+            No legislation found. <a
+              :href="`/search/?q=${q}`"
```

Comment:
`q` must be URL encoded, otherwise if it has "=" or "&" or "?" it'll break

Reply:
done